nengo_bones/templates/pkg/version.py.template

```diff
@@ -0,0 +1,33 @@
+# Automatically generated by nengo-bones, do not edit this file directly
+
+"""{{ project_name }} version information.
+
+{% if type == "semver" %}
+We use semantic versioning (see http://semver.org/)
+{% elif type == "calver" %}
+We use calendar versioning (see https://calver.org/)
+{% endif %}
+and conform to PEP440 (see https://www.python.org/dev/peps/pep-0440/).
+'.dev0' will be added to the version unless the code base represents
+a release version. Release versions are git tagged with the version.
+"""
+
+{% if type == "semver" %}
+version_info = ({{ major }}, {{ minor }}, {{ patch }})
+{% elif type == "calver" %}
+from datetime import date
+
+today = date.today()
+version_info = (today.year - 2000, today.month)
+{% endif %}
+
+name = "{{ pkg_name | replace("_", "-") }}"
+dev = {% if release %}None{% else %}0{% endif %}
+
+
+# use old string formatting, so that this can still run in Python <= 3.5
+# (since this file is parsed in setup.py, before python_requires is applied)
+version = "{v}{dev}".format(
+    v=".".join(str(v) for v in version_info),
+    dev=(".dev%d" % dev) if dev is not None else "",
+)
```

Comment:
Could we add the copyright string as used e.g. [here](https://github.com/nengo/nengo/blob/32acd92bc97669beda417e5006b72eb90fde3341/nengo/__init__.py#L53)? That way, we can update that dynamically with the year and don't have to manually do it each year. Could also add the `__license__` string that appears there, too, I guess.

Reply:
Added the copyright string. I didn't do the `__license__` string since it seemed like kind of a stretch to put that in `version.py`, and it will change very rarely so it doesn't seem as important to template it.